### PR TITLE
feat: added home page

### PR DESCRIPTION
**Pull request by suryxks**
Added landing page for E-commerce site

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Nectar Decor</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://nectarui.netlify.app/components/css/components.css" />
+</head>
+
+<body>
+    <nav class="header-nav">
+        <div class="brand">
+            <a href="#">
```
Comment (AbhishekkGautam) [inline index.html]: we can put '/' here to redirect the user to the home page.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        <div class="actions">
+            <a href="#" class=""><img src="/assets/heart.svg" class="action"></a>
+            <a href="#" class=""><img src="/assets/shopping-bag.svg" class="action"></a>
+            <a href="#" class=""><img src="/assets/user (1).svg" class="action"></a>
+        </div>
+    </nav>
+    <h1 class="heading-xl text-center">Categories</h1>
+    <section class="categories">
+        <div class="categories-card">
+            <a href=""><img src="./assets/plants-category-ecom.jpg"></a>
```
Comment (AbhishekkGautam) [inline index.html]: we can provide 'alt' to the image for better accessibility.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    color: var(--cta);
+    width: 100%;
+    position: relative;
+    top: -5rem;
+    text-align: center;
+}
+
+.categories-card {
+    border: 2px solid var(--border);
+    /* box-shadow:
```
Comment (AbhishekkGautam) [inline style.css]: we can remove unused CSS styling.

**Review by AbhishekkGautam — APPROVED** [review]
everything else looks good to me.

**Review by Swapnilkota108 — APPROVED** [review]
good to go!

**Review by techieshubham30 — APPROVED** [review]
Looks good!

File: style.css (hunk 3 above)
Comment (suryxks) [reply style.css]: Thank you for pointing it out abhishek , i have made the changes will merge it

**Pull request merged**